docs/pages/indexer.mdx

```diff
@@ -47,14 +47,191 @@ They are written under the assumption you are using `anvil` for your test chain,
 
 ### Using the indexer with TypeScript
 
+1. Create these files in a previously empty directory:
+
+   ```json filename="package.json" showLineNumbers copy
+   {
+     "name": "client",
+     "version": "0.0.0",
+     "private": true,
+     "license": "MIT",
+     "type": "module",
+     "scripts": {
+       "indexerClient": "npx ts-node --esm index.ts"
```

Comment:
no need for `npx` here if you have `ts-node` as a dep

also we tend to use `tsx` as it has slightly better ergonomics

```suggestion
       "start": "tsx index.ts"
```